README.md

```diff
@@ -136,6 +136,28 @@ If you are running webpack programmatically and wants to force this behaviour yo
   ...
 ```
 
+#### RuntimeOptions (default `undefined`)
+
+This allows you to control aspects of how `elm-make` runs with [GHC Runtime Options](https://downloads.haskell.org/~ghc/7.10.1/docs/html/users_guide/runtime-control.html).
+
+The 0.18 version of `elm-make` supports a limited set of those options, the most useful of which is
+for profiling a build.  To profile a build use the settings `runtimeOptions: '-s'`, which will print
+out information on how much time is spent in mutations, in the garbage collector, etc.
+
+If you notice your build spending a lot of time in the garbage collector, you can likely optimize it
+with some additional flags to give it more memory, `e.g. -A128M -H128M -n8m`.
```

Comment:
This involves building a new elm-make binary. I don't think documenting that is useful without stating so.

Reply:
Good point, I added a note in about it with c9a5658264d5425fe5599b911340e1bf701c8864